src/util.ts

```diff
@@ -744,10 +745,17 @@ export class Util {
     /**
      * Does the string appear to be a uri (i.e. does it start with `file:`)
      */
-    private isUriLike(filePath: string) {
+    public isUriLike(filePath: string) {
         return filePath?.indexOf('file:') === 0;// eslint-disable-line @typescript-eslint/prefer-string-starts-ends-with
     }
 
+    /**
+     * Gets a URI string for a file's srcPath
+     */
+    public getFileUri(file: BscFile) {
```

Comment:
Do we need this function? Isn't it almost just as easy to call `util.pathToUri(file?.srcPath)`? Seems like an extra function hop for fairly low benefit.